doc/RUNNING_TESTS.md

```diff
@@ -120,6 +120,24 @@ executing scenarios by name, e.g. `"cucumber:all[-n 'Adding an issue link']"`.
 Like with spaces in `cucumber:custom` arguments, task name and arguments
 have to be enclosed in quotation marks.
 
+#### Running cucumber features without rake
+
+Running cucumber features without going through `rake` is possible by using
+the following command
+
+`cucumber -r features features/my/path/to/cucumber.feature`
```

Comment:
This will only work for the core. The plugin features are not supported that way because the plugin's step definitions will not be loaded via this command. 

This however can still be covered by the rake task we have, so it shouldn't be a problem.